study-builder/studies/testboston/patches/sample-collect-message.conf

```diff
@@ -0,0 +1,26 @@
+{
+  "block": {
```

Comment:
This is used for patching up existing study running in deployed environments, so using a separate file here.